src/scoreboard.js

```diff
@@ -0,0 +1,12 @@
+var models = require('./models');
+
+module.exports = function (req, res, next){
+  models.User.findAll({
+    attributes: ['name', 'score'],
+    order: [ [ 'score', 'DESC' ] ]
+  }).then(scores => {
+    if (scores) res.send(scores);
+    else res.sendStatus(400);
+    next();
+  })
```

Comment:
add the `.catch(next)` here. Basically it does `.catch(err => next (err)` and calls the error handling middleware with `err`